handbook/people.md

```diff
@@ -24,6 +24,7 @@ DRIs help us collaborate efficiently by knowing exactly who is responsible and c
 | Lead engineering standup and sprint planning | Zach Wasserman* |
 | Customer Slack channels | Tony Gauda |
 | Customer renewals | Tony Gauda |
+| Documentation quality | Tony Gauda, Kathy Satterlee, Kelvin Omereshone |
```

Comment:
```suggestion
| Documentation quality | Mike Thomas |
```
```suggestion
| Documentation quality | Tony Gauda, Kathy Satterlee, Kelvin Omereshone |
```

Reply:
Thanks for adding this, @tgauda. DRI should be one person, so I've put myself down as I'm looking after dev advocates for the time being 👍🏻

Reply:
Makes sense. Thanks for updating it!